src/store/messagesStore.js

```diff
@@ -542,6 +551,31 @@ const actions = {
 			return
 		}
 
+		if (message.parent && message.systemMessage === 'message_edited') {
+			// If parent message is presented in store already, we update it
+			const parentInStore = context.getters.message(message.token, message.parent.id)
+			if (Object.keys(parentInStore).length !== 0) {
+				context.commit('addMessage', message.parent)
+				return
+			}
+
+			// Update message in store
+			const messageUpdated = message.parent
```

Comment:
This code below shouldn't be needed, as you just updated parent in the store with all these parameters